### ISSUE TITLE
Dashed lines

### ISSUE THREAD

**Issue by 212361828** [PredixDev/px-vis-timeseries#52]
We require the ability to represent lines in dash format.  This is available for threshold lines but not for actual series.

From a UX perspective the dashed line would also need to be represented in acouple other places for consistency.

I think I have an idea how to go about it so PR en route.

Detailed requirements
- Needs to represent series with varying dash patterns
- Color and dash pattern must be represented in legend (register) and series axis
- Dash pattern must also render to canvas
- Dash pattern must also be reflected in navigator (probaly less important but what the hell)
- Probably good to use 'dashPattern' as the property name to stay consistent with threshold-config

**Comment by nonmetalhail** [comment]
Great. Look forward to the PR

**Comment by 212361828** [comment]
PR [90](https://github.com/PredixDev/px-vis/pull/90) created against px-vis.

**Comment by 212361828** [comment]
@nonmetalhail , not sure what the process is to point px-vis-timeseries at the latest version of px-vis.  Is that something I should be pushing or will that just happen?  Thanks!

**Comment by nonmetalhail** [comment]
I'm currently in Austria, so it was midday for me  ;)

I would tag it, but Travis is erroring out due to an unrelated issue. When it can run again, I'll confirm that it all works and tag it. 

FYI: I made a couple adjustments to the bar size. It got narrower than it used to be so I restored the original size. Dont know if that was an intentional change, but we've gotta keep with our spec. Also, you used some template literals in the register which I didnt notice until the tests were failing in browsers other than Chrome and FF. So I changed that, but then Travis stopped working. So hopefully everything will pass now. At any rate, I'll tag once it does.

Thanks again. Been wanting to do this for a while; just never had the time :)
